mainapp/urls.py

```diff
@@ -5,7 +5,11 @@
     path('genre_books/<genre>', views.genre_books, name='genre_books'),
     path('explore_books/', views.explore_books, name='explore_books'),
     path('book_recommendations/', views.book_recommendations, name='book_recommendations'),
-    path('library/read_books', views.read_books, name='read_books')
+    path('library/read_books', views.read_books, name='read_books'),
+    path('save/<int:id>', views.save, name='save'),
+    path('remove/<int:id>', views.remove, name='remove'),
+    path('library/to_read', views.SaveList, name='to_read')
```

Comment:
`library/saved_books`  would be more clear what u say @Praful932

Reply:
yea lets make it `saved_books`